app/Views/admin/emails/Emails.php

```diff
@@ -61,11 +62,16 @@
                                             </div>
                                         </td>
                                         <td>
-                                            <div class="text-<?= ($email['emailTypeId'] === '1') ? 'warning' : (($email['emailTypeId'] === '2') ? 'info' : 'danger'); ?> text-wrap">
+                                            <div class="text-<?= ($email['emailTypeId'] === '1') ? 'warning-emphasis' : (($email['emailTypeId'] === '2') ? 'info-emphasis' : 'success-emphasis'); ?> text-wrap">
                                                 <?= ($email['emailTypeId'] === '1') ? 'Contacto' : (($email['emailTypeId'] === '2') ? 'Soporte técnico' : 'Información producto'); ?>
                                             </div>
 
                                         </td>
+                                        <td>
+                                            <div class="d-flex align-items-center fw-medium">
+                                                <p class="text-wrap"><?= $email['createdAt']; ?></p>
```

Comment:
I think you should format the date better as "day/month/year"
```suggestion
  <p class="text-wrap"><?= date('j/m/Y', strtotime($email['createdAt']))?></p>
```

Reply:
Maybe for this, as you have the hour, you can set the date on that format and the hour like this:
```suggestion
<p class="text-wrap"><?= date('j/m/Y, g:i a', strtotime($email['createdAt']))?></p>
```

In case you want more date format recomendations, you have thie reference @FernandoJoachin: [date on php](https://www.php.net/manual/es/function.date.php)